examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -109,6 +112,7 @@ export default class App extends PureComponent {
       selectedMapStyle: INITIAL_MAP_STYLE,
       debugOptions: {
         minimap: true,
+        minimapViewport: false,
```

Comment:
- Move this into a constant `INITIAL_STATE` at the top of the file, as the constructor here is becoming very long?
- Add a comment above each setting? E.g. it is not immediately clear what the difference between e.g. `minimap` and `minimapViewport` is.